core/src/main/java/io/confluent/connect/storage/StorageSinkConnectorConfig.java

```diff
@@ -120,6 +131,20 @@ public static ConfigDef newConfigDef(ConfigDef.Recommender formatClassRecommende
       );
 
       configDef.define(
+          AVRO_CODEC_CONFIG,
```

Comment:
Its importance is low and there are other configs with higher importance. Lets add it somewhere in the same group but further below